docs/pipelineruns.md

```diff
@@ -627,6 +627,20 @@ Task Runs:
         foo
 ```
 
+The name of the `TaskRuns` and `Runs` owned by a `PipelineRun`  are univocally associated to the owning resource.
+If a `PipelineRun` resource is deleted and created with the same name, the child `TaskRuns` will be created with the
+same name as before. The base format of the name is `<pipelinerun-name>-<pipelinetask-name>`. The name may vary
+according the logic of [`kmeta.ChildName`](https://pkg.go.dev/github.com/knative/pkg/kmeta#ChildName).
+
+Some examples:
+
+| `PipelineRun` Name       | `PipelineTask` Name          | `TaskRun` Name    |
+|--------------------------|------------------------------|-------------------|
+| pipeline-run             | task1                        | my-pipeline-task1 |
```

Comment:
NIT: will the `taskRun` name be `pipeline-run-task1` instead of `my-pipeline-task1`?

Reply:
Nice catch, thank you!